libcomposefs/lcfs-writer.c

```diff
@@ -471,9 +471,16 @@ struct lcfs_node_s *lcfs_node_new(void)
 		errno = ENOMEM;
 		return NULL;
 	}
-
+	
+	memset( node, 0, sizeof(struct lcfs_node_s));
+	node->payload = NULL;
+	node->content = NULL;
 	node->ref_count = 1;
 	node->inode.st_nlink = 1;
+	node->delayedDigestCalculation = false;
```

Comment:
Naming scheme in this project for member variables is `snake_case`, not `camelCase`.

Reply:
I have corrected it here and in rest of my changes.